rtil/src/native/mod.rs

```diff
@@ -27,7 +27,7 @@ mod tick;
 mod app;
 mod memory;
 mod hud;
-mod uworld;
+pub mod uworld;
```

Comment:
This is only public because `TimeOfDay` is accessed from the outside. Instead of exposing all of the uworld to the world (pun intended), you can `pub use` the item itself just like has been done for `APawn`, `UWorld` and `UGameplayStatics`.